src/main/java/seedu/algobase/model/ModelManager.java

```diff
@@ -51,7 +52,8 @@ public ModelManager(ReadOnlyAlgoBase algoBase, ReadOnlyUserPrefs userPrefs) {
         this.userPrefs = new UserPrefs(userPrefs);
         filteredProblems = new FilteredList<>(this.algoBase.getProblemList());
         filteredTags = new FilteredList<>(this.algoBase.getTagList());
-        sortedProblems = (new SortedList<>(filteredProblems));
+        sortedProblems = new SortedList<>(filteredProblems);
+        sortedProblems.setComparator(SortCommand.PROBLEM_NAME_COMPARATOR);
```

Comment:
Why added this line?

Reply:
To give the problem list a default comparator. If not the order of the problems is a bit indeterministic.